tools/cli/go-whisk-cli/commands/activation.go

```diff
@@ -365,7 +365,7 @@ var activationPollCmd = &cobra.Command{
                 Skip: 0,
             }
 
-            activations, _, err := client.Activations.List(options)
+            activations, _, err := Client.Activations.List(options)
```

Comment:
Debug message below should be `Client.Activations.List` instead of `client.Activations.List`. Should also display options as the argument in the debug message.